### PR TITLE
DOCS-2295 Vulnerability search and filters

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/search-and-filters/use-search-vuln.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+[al-vuln-comp]:           ../../attacks-vulns-list.md#vulnerable-component
+[al-ssrf]:                ../../attacks-vulns-list.md#serverside-request-forgery-ssrf
+[al-csrf]:                ../../attacks-vulns-list.md#cross-site-request-forgery-csrf
+[al-vuln-component]:      ../../attacks-vulns-list.md#vulnerable-component
+[ssti-injection]:         ../../attacks-vulns-list.md#serverside-template-injection-ssti
+[al-weak-jwt]:            ../../attacks-vulns-list.md#weak-jwt
+
+# Vulnerability Search and Filters
+
+Wallarm provides convenient methods for searching detected vulnerabilities. In the **Vulnerabilities** section of Wallarm Console, the following search methods available:
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/search-and-filters/use-search-vuln.md:20]: make the overview good for SEO. do not say practically the same thing in 2 sentences

File: mkdocs-4.8.yml [hunk 2]
```diff
@@ -58,6 +58,7 @@ nav:
     - Vulnerability Assessment:
       - Detecting Vulnerabilities: about-wallarm/detecting-vulnerabilities.md
       - Managing Vulnerabilities: user-guides/vulnerabilities.md
+      - Search and Filters: user-guides/search-and-filters/use-search-vuln.md
```
Comment (AnastasiaTWW) [inline mkdocs-4.8.yml:61]: is the Home page update needed?

File: docs/latest/user-guides/search-and-filters/use-search-vuln.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+[al-xxe]:                 ../../attacks-vulns-list.md#attack-on-xml-external-entity-xxe
+[al-ldapi]:               ../../attacks-vulns-list.md#ldap-injection
+[al-infoleak]:            ../../attacks-vulns-list.md#information-exposure
+[al-vuln-comp]:           ../../attacks-vulns-list.md#vulnerable-component
+[al-ssrf]:                ../../attacks-vulns-list.md#serverside-request-forgery-ssrf
+[al-csrf]:                ../../attacks-vulns-list.md#cross-site-request-forgery-csrf
+[al-vuln-component]:      ../../attacks-vulns-list.md#vulnerable-component
+[ssti-injection]:         ../../attacks-vulns-list.md#serverside-template-injection-ssti
+[al-weak-jwt]:            ../../attacks-vulns-list.md#weak-jwt
+
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/search-and-filters/use-search-vuln.md:17]: do we need to get rid of vulnerability search description in our existing article on event search?

File: mkdocs-4.8.yml (hunk 2 above)
Comment (MaximBashurov) [reply mkdocs-4.8.yml:61]: No, it has only a general "Vulnerability Assessment" link.

File: docs/latest/user-guides/search-and-filters/use-search-vuln.md (hunk 3 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/search-and-filters/use-search-vuln.md:17]: The article on event search does not contain anything describing vulnerability search.

**Pull request merged**